src/sybil/column_store_io.go

```diff
@@ -659,12 +659,6 @@ func (tb *TableBlock) unpackSetCol(dec FileDecoder, info SavedColumnInfo) {
 		}
 	} else {
 		for r, v := range into.Values {
-			curSet, ok := records[r].SetMap[colID]
-			if !ok {
-				curSet = make(SetField, 0)
-				records[r].SetMap[colID] = curSet
-			}
-
```

Comment:
we should likely test this by inducing a high cardinality set codepath. can do that by creating a set column where each set includes a string that is counting upwards to 65,000.